spark/src/test/scala-spark-master/shims/logging/DeltaStructuredLoggingSuite.scala

```diff
@@ -44,10 +44,22 @@ import com.fasterxml.jackson.databind.ObjectMapper
 import com.fasterxml.jackson.module.scala.DefaultScalaModule
 import org.apache.logging.log4j.Level
 
+import org.apache.spark.internal.Logging
+
 class DeltaStructuredLoggingSuite extends DeltaStructuredLoggingSuiteBase {
   override def className: String = classOf[DeltaStructuredLoggingSuite].getSimpleName
   override def logFilePath: String = "target/structured.log"
 
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+    Logging.enableStructuredLogging()
```

Comment:
Structured logging is not enabled by default (anymore), so we have to enable it.